api/src/pcapi/routes/backoffice_v3/collective_offers/blueprint.py

```diff
@@ -318,5 +322,116 @@ def batch_reject_collective_offers() -> utils.BackofficeResponse:
         flash("L'un des identifiants sélectionnés est invalide", "danger")
         return redirect(request.referrer or url_for("backoffice_v3_web.collective_offer.list_collective_offers"), 303)
 
-    _batch_validate_or_reject_collective_offers(OfferValidationStatus.REJECTED, form.object_ids_list)
+    _batch_validate_or_reject_collective_offers(offer_mixin.OfferValidationStatus.REJECTED, form.object_ids_list)
     return redirect(request.referrer or url_for("backoffice_v3_web.collective_offer.list_collective_offers"), 303)
+
+
+def _is_collective_offer_price_editable(collective_offer: educational_models.CollectiveOffer) -> bool:
+    # if the offer has no stock it has no price therefore its price cannot be updated
+    if not collective_offer.collectiveStock:
+        return False
+
+    # cannot update an offer's price while the cashflow generation script is running
+    if app.redis_client.exists(finance_conf.REDIS_GENERATE_CASHFLOW_LOCK):  # type: ignore [attr-defined]
+        return False
+
+    # cannot update an offer's stock if it is already a processed pricing
```

Comment:
```suggestion
    # cannot update an offer's stock if it already has a processed pricing
```
C'est une suggestion, je suis pas assez certain du contexte pour savoir si c'est plus adapté